packages/manager/modules/config/src/environment/environment.service.js

```diff
@@ -11,6 +11,7 @@ export default class EnvironmentService {
     this.userLocale = findAvailableLocale(detectUserLocale(), this.region);
     this.version = null;
     this.user = {};
+    this.baseUrls = {};
```

Comment:
maybe ```applicationURLs``` would describe the attribute better, what do you think?

Reply:
Make sense, it's applications bases urls.